openid-federation-1_0.xml

```diff
@@ -6733,42 +6736,72 @@ HTTP/1.1 302 Found
 	  as the <spanx style="verb">crit</spanx> value.
 	  Use of this claim is OPTIONAL.
 	</t>
+	<t>
+	  This claim is used in this specification in <xref target="entity-statement"/>
+	  to identify claims not defined by this specification
+	  that MUST be understand and processed
```

Comment:
Again would not 'understood' be better ?

Reply:
Thanks, fixed!